packages/tsconfig/package.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "tsconfig",
```

Comment:
thoughts on `@zoralabs/tsconfig` even if it's never published would prevent collisions.